src/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -59,6 +65,8 @@
     private static final String KEY_TOPS = "TOPS";
     private static final String KEY_HEIGHT_CELL = "HEIGHT_CELL";
     private static final String KEY_EMPTY_LIST_MESSAGE = "EMPTY_LIST_MESSAGE";
+    private static final String GRID_COLUMNS = "gridColumns";
+    private ScaleGestureDetector SGD = null;
```

Comment:
since its is not a static it should be mSGD or even better mScaleGestureDetector, SGD is too enigmatic